python/chemiscope/jupyter.py

```diff
@@ -194,7 +194,7 @@ def _is_running_in_notebook():
 
     try:
         shell = get_ipython().__class__.__name__
-        if shell == "ZMQInteractiveShell":
+        if shell == "ZMQInteractiveShell" or "Interpreter":
```

Comment:
that does not do what you think it does ^_^. This code evaluate "Interpreter" to a bool (`True`), and then evaluate the `or`.

```suggestion
        # ZMQInteractiveShell is the standard Jupyter Kernel
        # Interpreter is used by pyiodide
        if shell in ["ZMQInteractiveShell", "Interpreter"]:
```